src/templates/record.vue

```diff
@@ -149,6 +149,9 @@ limitations under the License.
             ns() {
                 this.handleNsRegistrar()
             },
+            registrar() {
```

Comment:
Looks to be a dupe of the above method?

Reply:
It is, there's not really a better way to do it though. Each watch needs to be its own function.

Reply:
Should the methods they call not be split apart then so that they are fully independent?